### PR TITLE
Delete task

### PR DESCRIPTION
**Pull request by DanielGSoftware**
## Summary by CodeRabbit

- **New Features**
	- Introduced a command for deleting tasks, accessible via command line.
	- Enhanced task management functionality by allowing users to delete tasks through the application interface.
- **Bug Fixes**
	- Improved error handling for attempts to delete non-existing tasks, providing clear feedback to users.
- **Tests**
	- Added comprehensive tests to validate the task deletion functionality and ensure robustness against edge cases.

### CODE REVIEW

File: src/Application/Tasks/Delete/DeleteTaskService.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    {
+        $task = $this->taskWriteRepository->getById($taskId);
+
+        if (! $task) {
+            throw new TaskNotFoundException("Can't delete task: task not found");
+        }
+
+        $this->taskWriteRepository->delete($taskId);
+    }
+}
```
[the remark this answers is not in the document]
Comment (DanielGSoftware) [reply src/Application/Tasks/Delete/DeleteTaskService.php:24]: Isn't this covered in DeleteTaskTest?

[the remark this answers is not in the document]
Comment (DanielGSoftware) [reply src/Application/Tasks/Delete/DeleteTaskService.php:24]: @coderabbitai
